### PR TITLE
Add getrandom

### PR DESCRIPTION
**Pull request by richsalz**
Fixes the "random number" discussion in #59.

### CODE REVIEW

File: Awesome_Rust_Cryptography.md [hunk 1]
```diff
@@ -545,6 +545,9 @@ by large quantum computers.
 - [rand](https://github.com/rust-random/rand) Rust library for random number
  generation.
 
+- [getrandom](https://docs.rs/getrandom/0.2.3/getrandom/) Interfave to the
```
Comment (tarcieri) [inline Awesome_Rust_Cryptography.md:548]:
```suggestion
- [getrandom](https://docs.rs/getrandom/0.2.3/getrandom/) Interface to the
```

Comment (richsalz) [reply Awesome_Rust_Cryptography.md]: fixup commit pushed.

**Comment by tarcieri** [comment]
Generally I'd suggest people not use `getrandom` directly, but rather consume it through `rand_core::OsRng`.

Curious what @newpavlov thinks

**Comment by richsalz** [comment]
It's just a step on the FIPS journey :)  Next step is being able to use it to seed DRBG. :)

**Pull request merged**